### PR TITLE
WSO 63 google analytics tabbing

### PR DESCRIPTION
**Pull request by TDRoden**

### CODE REVIEW

File: sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js [hunk 1]
```diff
@@ -36,6 +36,12 @@ var ucgy_analytics;
             // We had to duplicate this code in order to manage secondary trackers more effectively
             // Catch the closest surrounding link of a clicked element.
             $(document.body).bind("mousedown keyup touchstart", function(event) {
+
+                // If it is a key press, we are only concerned with tracking 'enter' key interactions
+                if ((event.type == "keyup") && (event.which != 13)) {
+                    return;
+                }
+
```
Comment (SandyVallee) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: where is the code that does throw the ga event if its an enter to launch the link?  should there be an else statement to this?

Comment (SandyVallee) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: I see that it is still launching the ga event on an enter key press so I'm sure its somewhere else in the file=) You can ignor this.

Comment (TDRoden) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: If it is an 'enter' keyboard event, it just continues through to the next block and is processed along with the mousedown and touchstart events. All other keyboard events are discarded/ignored. I feel an else statement here complicates things as you'd also have to account for mousedown and touchstart.

Comment (SandyVallee) [inline sites/all/themes/ucgy_theme/resources/js/ucgy_analytics.js]: sounds good =)

**Pull request merged**